user/session.go

```diff
@@ -62,6 +62,7 @@ type AccessDefinition struct {
 	AllowedURLs       []AccessSpec            `bson:"allowed_urls" json:"allowed_urls" msg:"allowed_urls"` // mapped string MUST be a valid regex
 	RestrictedTypes   []graphql.Type          `json:"restricted_types" msg:"restricted_types"`
 	AllowedTypes      []graphql.Type          `json:"allowed_types" msg:"allowed_types"`
+	EnableAllow       bool                    `json:"enable_allow" msg:"enable_allow"`
```

Comment:
Do we really need flag here?
Or just check for non empty Array instead?

Reply:
`enable_allow` flag is proposed by the ticket's author. I followed the description. What if the user creates a policy or key with both non-empty allow and block list arrays? May the user want to switch between those two options easily? What do you think @kofoworola and @Aenimus?

Reply:
Let me think about it a little, Burak. Or we should do a call together?

Reply:
I think we can discuss it after the daily.

Reply:
Well if you have both allow list takes priority, and from logic point of you it all makes sense